CHANGELOG.md

```diff
@@ -8,6 +8,81 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
 
 _No documentation available about unreleased changes as of yet._
 
+## [2.1.0] - 2019-04-08
+
+### Added
+- New `WordPress.PHP.IniSet` sniff to the `WordPress-Extra` ruleset.
+    This sniff will detect calls to `ini_set()` and `ini_alter()` and warn against their use as changing configuration values at runtime leads to an unpredictable runtime environment, which can result in conflicts between core/plugins/themes.
+    - The sniff will not throw notices about a very limited set of "safe" ini directives.
+    - For a number of ini directives for which there are alternative, non-conflicting ways to achieve the same available, the sniff will throw an `error` and advise using the alternative.
+- `doubleval()`, `count()` and `sizeof()` to the list of functions which unslash and sanitize `Sniff::$unslashingSanitizingFunctions`.
```

Comment:
Is this correct? The explanation made need more ...explaining, as at first glance, `count()` et al does not unslash or sanitize (though it does change it to a known safe type).

Reply:
I've added a little more explanation to this bullet point. Better ?